packages/components/src/assets/TagIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FC } from 'react'
+import React from 'react'
+
+// Icon from Phosphor Icons
+const TagIcon: FC = () => (
```

Comment:
Why do we can it `TagIcon` instead of only `Tag` ? 🤔

Reply:
That's because we already have a `Tag` component with this same name, and they conflict while exporting.